tools/ci_build/github/azure-pipelines/linux-cpu-minimal-build-ci-pipeline.yml

```diff
@@ -179,15 +179,23 @@ jobs:
 
 
   - task: AzureCLI@2
-    displayName: 'Azure CLI'
-    condition: and (succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI'), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    displayName: 'Publish binary size data'
+    condition: and(succeededOrFailed(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI'), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
What if the build failed, will this still be executed?

Reply:
If build failed (not failed at where the binary size is compared), then the `binary_size_data.txt` will not be generated, so this step will fail also